.circleci/config.yml

```diff
@@ -238,6 +247,8 @@ jobs:
           key: dependency-cache-v1-{{ checksum "yarn.lock" }}
           paths:
             - node_modules/
+            - .yarn/cache/
+            - .yarn/install-state.gz
```

Comment:
Hmm, we have never cached this in our library workflows. What is this for?

Reply:
> .yarn/install-state.gz is an optimization file that you shouldn't ever have to commit. It simply stores the exact state of your project so that the next commands can boot without having to resolve your workspaces all over again

Idk if that means it helps at all in our situation but maybe it *would* help in a monorepo?

Reply:
Ah, hmm. I wonder what it's storing the state _of_. This might cause problems if we drop `node_modules` from the cache.

Reply:
Ill explore if it makes any impact on CI times and drop it if it does not